tests/unittests/GraphOptzTest.cpp

```diff
@@ -432,40 +440,43 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvButConvReused) {
 }
 
 TEST_F(GraphOptz, optimizeBatchNormAfterConvButVarReused) {
-  Node *A =
+  auto *A =
       mod_.createPlaceholder(ElemKind::FloatTy, {1, 10, 20, 3}, "A", false);
-  auto *filter =
-      mod_.createPlaceholder(ElemKind::FloatTy, {16, 5, 5, 3}, "filter", true);
-  auto *bias = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "bias", true);
-
-  ConvolutionNode *CV = F_->createConv(
-      "conv", A, filter, bias,
-      mod_.uniqueType(ElemKind::FloatTy, {1, 10, 20, 16}), 5, 1, 2, 1);
-  auto *beta = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "beta", true);
-  auto *gamma = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "gamma", true);
-
-  auto *mean = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "mean", false);
-  auto *var = mod_.createPlaceholder(ElemKind::FloatTy, {16}, "var", false);
 
-  Node *BN = F_->createBatchNormalization("batch", CV, beta, gamma, mean, var,
-                                          3, 0.0001, 0.9);
-  SaveNode *ret = F_->createSave("ret", BN);
-  SaveNode *filterSave = F_->createSave("filterSave", CV->getFilter());
+  ConvolutionNode *CV = F_->createConv(bindings_, "conv", A, 16, 5, 1, 2, 1);
+  Node *BN =
+      F_->createBatchNormalization(bindings_, "batch", CV, 3, 0.0001, 0.9);
+  F_->createSave("ret", BN);
+  F_->createSave("filter", CV->getFilter());
```

Comment:
Thanks for simplifying the test case construction code! Can you include screenshots of the original graph before and after your simplification to make sure they are the same? You can dump a dot graph using `Function::dumpDAG` and render it using `graphviz`.

Reply:
I'm not sure that this does preserve structure, but after the change the test is closer to testing what it actually purports to test. Also all of the other tests here setup their convs in this way.

Reply:
I'm pretty sure it does. The only difference between the old and the new method is that all of the `Placeholders` for the `Convolution` and `BatchNormalization` nodes are created inside `Function::createXXX` instead of in the scope of the test. 

It's easy to verify - just print out the graph structure using `Function::dumpDAG` before and after your change and examine it using `graphviz`. It is a good idea to include it in the test plan too.

Reply:
included below